packages/angular/src/generators/host/files/src/main.server.ts__tmpl__

```diff
@@ -39,7 +39,7 @@ export function app(): express.Express {
   );
 
   // All regular routes use the Universal engine
-  server.get('*', (req, res) => {
+  server.get('*', (req: any, res: any) => {
```

Comment:
Build fails due to Typescript errors

Reply:
These TS type errors are very interesting. They only arise with pnpm v7. I tried with pnpm 6 and it's all good. Also, the IDE doesn't report such an issue, even when the packages are installed with pnpm v7.

That said, by adding `@types/express` the types are inferred correctly and therefore, we don't need to add `any`.

Reply:
You are correct, adding types solved it